### PR TITLE
Maryland Content Updates 2024

### PR DESCRIPTION
**Pull request by squanto**
## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-542

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Updates all MD content 
- Adds sms terms oage that has to be accepted to continue

## How to test?
Step through MD flow

## Screenshots (for visual changes)
Screenshot 2024-12-04 at 5 07 43 PM

### CODE REVIEW

File: app/views/state_file/questions/md_review/edit.html.erb [hunk 1]
```diff
@@ -123,8 +131,7 @@
 
       <div class="spacing-below-25">
         <p class="text--bold spacing-below-5"><%= t("state_file.general.md_nonrefundable_local_earned_income_tax_credit") %></p>
-        <!--        <p><%#= number_to_currency(current_intake.calculator.line_or_zero(:MD502_LINE_29)) %></p>-->
-        WIP
+        <p><%= number_to_currency(current_intake.calculator.line_or_zero(:MD502_LINE_29)) %></p>
       </div>
```
Comment (tahsinaislam) [inline app/views/state_file/questions/md_review/edit.html.erb:135]: [pebble] theres a todo for this in the md review spec (spec/controllers/state_file/questions/md_review_controller_spec.rb) to add the tests for these lines that you shows now

**Review by tahsinaislam — APPROVED** [review]
would just add the tests for the md review page for the lines you show now but otherwise lgtm!

**Review by jnf — APPROVED** [review]
whew. lot in this one. lgtm tho.

File: app/views/state_file/questions/md_review/edit.html.erb (hunk 1 above)
Comment (squanto) [reply app/views/state_file/questions/md_review/edit.html.erb]: hmm i'm seeing that spec spec running fully (not sure how it's passed before now)

**Pull request merged**